llvm/lib/ExecutionEngine/RuntimeDyld/RuntimeDyldCOFF.h

```diff
@@ -15,10 +15,6 @@
 
 #include "RuntimeDyldImpl.h"
 
-#define DEBUG_TYPE "dyld"
```

Comment:
This header code does not seems to use anything like LLVM_DEBUG which needs the DEBUG_TYPE to be set.

Reply:
This is probably not worth touching. See https://discourse.llvm.org/t/rfc-removing-mcjit-and-runtimedyld/80464

Reply:
Thanks. Taking a cursory look, it seems it's not yet decided when the code will be removed. As such, might to ok to commit it. But if there is a push-back, I can undo the changes in some files.